common/install_android.sh

```diff
@@ -104,3 +104,83 @@ export PATH="${GRADLE_HOME}/bin/:${PATH}"
 echo "PATH:${PATH}"
 
 gradle --version
+
+rm -rf /tmp/gradledeps
+
+_gradledeps=/tmp/gradledeps
+
+rm -rf $_gradledeps
+
+cat <<EOF >"$_gradledeps/AndroidManifest.xml"
+<manifest package="org.pytorch" />
+EOF
+
+cat <<EOF >"$_gradledeps/build.gradle"
+buildscript {
+    ext {
+        minSdkVersion = 21
+        targetSdkVersion = 28
+        compileSdkVersion = 28
+        buildToolsVersion = '28.0.3'
+
+        coreVersion = "1.2.0"
+        extJUnitVersion = "1.1.1"
+        runnerVersion = "1.2.0"
+        rulesVersion = "1.2.0"
+        junitVersion = "4.12"
+    }
+
+    repositories {
+        google()
+        mavenLocal()
+        mavenCentral()
+        jcenter()
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.3.2'
+        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.0"
+        classpath "com.github.dcendents:android-maven-gradle-plugin:2.1"
+        classpath "org.jfrog.buildinfo:build-info-extractor-gradle:4.9.8"
+    }
+}
+
+repositories {
+    google()
+    jcenter()
+}
+
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion rootProject.compileSdkVersion
+    buildToolsVersion rootProject.buildToolsVersion
+
+    defaultConfig {
+        minSdkVersion minSdkVersion
+        targetSdkVersion targetSdkVersion
+    }
+
+    sourceSets {
+        main {
+            manifest.srcFile 'AndroidManifest.xml'
+        }
+    }
+}
+
+dependencies {
```

Comment:
Can we add fbjni-java-only to this?

Reply:
I tried with only fbjni-java-only - offline build does not work due to no deps in cache for:
'com.google.code.findbugs:jsr305:3.0.1'
'com.facebook.soloader:nativeloader:0.8.0'

if to add them explicitly - works.

I will try to understand why this happens